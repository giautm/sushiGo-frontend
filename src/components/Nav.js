import React from 'react';

const Nav = props => {
  return (
    <div className="navbar">
      <h1>SushiGo</h1>
    </div>
  );
};

export default Nav;
