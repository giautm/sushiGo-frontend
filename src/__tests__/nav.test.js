import React from 'react';
import Nav from '../components/Nav';
import renderer from 'react-test-renderer';

describe('Nav', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Nav />);
    expect(component).toMatchSnapshot();
  });
});
