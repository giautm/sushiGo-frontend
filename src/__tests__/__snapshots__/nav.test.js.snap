// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<div
  className="navbar"
>
  <h1>
    Sushi Go
  </h1>
</div>
`;
